src/platform/backends/qemu/qemu_mount_handler.cpp

```diff
@@ -38,54 +29,58 @@ namespace
 constexpr auto category = "qemu-mount-handler";
 } // namespace
 
-mp::QemuMountHandler::QemuMountHandler(const SSHKeyProvider& ssh_key_provider) : MountHandler(ssh_key_provider)
+namespace multipass
 {
-}
-
-void mp::QemuMountHandler::init_mount(VirtualMachine* vm, const std::string& target_path, const VMMount& vm_mount)
+QemuMountHandler::QemuMountHandler(QemuVirtualMachine* vm, const SSHKeyProvider* ssh_key_provider, std::string target,
+                                   const VMMount& mount)
+    : MountHandler{vm, ssh_key_provider, target, mount},
+      qemu_vm{vm},
+      // Create a reproducible unique mount tag for each mount. The cmd arg can only be 31 bytes long so part of the
+      // uuid must be truncated. First character of tag must also be alpabetical.
+      tag{QUuid::createUuidV3(QUuid(), QString::fromStdString(target))
```

Comment:
Not entirely sure, as I didn't make this decision.
@sharder996, I think you wrote this initially, right? Any reason to use a UUID v3 generated from the target? I don't really see a reason to tie the UUID to the target. Can't they be totally unrelated?

Reply:
It's generated from the target because each mount needs to have a unique tag. Since the mount target also needs to be unique, I used it as a seed to create the mount tag.

Reply:
I understand that the tag must be unique, but a non-seeded UUID should pretty much guarantee that. Nevertheless, I don't think it would be a bad idea to allow `mp::utils::make_uuid` to be seeded. So I'll add that :)

Reply:
Oh, the other reason I needed to seed the tag is because we need it in two places; once to pass into qemu when creating the process and the second when actually mounting.

Reply:
Ah, yes, indeed. Well, now the tag gets generated only once in the constructor, so I guess we can just skip the whole seed thing entirely and use `mp::utils::make_uuid` as is.